Makefile.dune

```diff
@@ -1,7 +1,7 @@
 # -*- mode: makefile -*-
 # Dune Makefile for Coq
 
-.PHONY: help voboot states world watch check quickbyte quickopt test-suite release apidoc ocheck ireport clean
+.PHONY: help voboot states world nogtk watch check quickbyte quickopt test-suite release apidoc ocheck ireport clean
```

Comment:
You were splitting this in another PR. I think it's a good idea to stop this line from growing forever.